ui/src/components/credentials/IssueCredential.tsx

```diff
@@ -70,9 +70,10 @@ export function IssueCredential() {
     [schemaID]
   );
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Ouch. Why?

Reply:
I wanted to be explicit about the schema being required by this function, but no prob, we will find out anyway. Removed.